docs/ccv1_image_security_design.md

```diff
@@ -0,0 +1,345 @@
+
+The signature claim file is what we usually call the container image signature file. Its directory structure and file name format are as follows:
+
+```
+.
+└── busybox@sha256=817a12c32a39bbe394944ba49de563e085f1d3c5266eb8e9723256bc4448680e
+    └── signature-1
+```
+
+The meaning of directory name is `image-name@digest-algo=digest-value`, multiple signature files can exist in the directory, which means that the same container image can have multiple signatures and be numbered and stored.
+
+### Image distribution
+
+After the encryption and signing are completed, we obtain two contents: the container image itself and the signature claim file, which are distributed and stored separately.
+
+- container image: pushed to the specified remote registry.
+
+- signature claim file: upload to the specified sigstore server.
+
+## Deployment of protected container image
+
+In the confidential container V1 design, security functions will be performed when pulling the container image, including checking the registry whitelist, verifying the container image signature, and decrypting the container image layer. The high-level process is as follows:
+
+1. Check whether the registry where the image is located is in policy.json's whitelist.
+
+2. Download the signature claim file of the container image according to the configuration of the sigstore file.
+
+3. Verify the signature of the container image according to the configuration of the policy.json file.
+
+4. Pull the container image layer by layer. If an encrypted container image layer is encrypted, decrypt it.
+
+After the image is pulled, the subsequent deployment process is consistent with that of the normal container image.
+
+### policy.json
+
+policy.json is a necessary configuration file for deploying images in the confidential container V1 design. Its location is specified in the /etc/containers/ directory of boot image rootfs. Its main functions are as follows:
+
+- Declare a whitelist of container images with registry granularity
+
+- Configure the public key to use when verifying the image signature
+
+Thus, policy.json is confidential information that must be controlled by users and cannot be read or modified by untrusted entities.
+
+If image-rs finds that there is no policy.json in the /etc/containers/ directory or If the policy.json file is empty, call the GetResource grpc service provided by the attestation-agent. After remote attestation, obtain it from the user controlled relying KBS through the encrypted channel.
+
+The format of the policy.json file follows the format and syntax defined in the [container/image policy.json](https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md) document. In the V1 design of the confidential containers, a safe and reasonable policy.json file should look like the following example: 
+
```

Comment:
This is good. I'm wondering if it would make sense to provide some sample policies like flexible which allows everything (`insecureAcceptAnything`), restrictive (for CC) etc.
image-rs functionality is generally usable feature (similar to api restriction feature) and I'm envisaging a scenario for development and otherwise where having sample policies will help.

Reply:
OK, I have added some other examples in the document. For more and more complete examples, please refer to [container/image policy.json](https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md)